netlify/package.json

```
@@ -2,6 +2,12 @@
   "name": "asciidoctor-docs-netlify",
   "description": "Asciidoctor Docs (Production and Deploy Previews)",
   "private": true,
+  "scripts": {
+    "clean-build": "npm run clean-install;npm run build-fetch",
+    "clean-install": "rm -rf node_modules/ .cache/ package-lock.json;npm i --cache=.cache/npm --no-optional",
+    "build-fetch": "node_modules/.bin/antora ../antora-playbook.yml --stacktrace --fetch",
+    "build": "node_modules/.bin/antora ../antora-playbook.yml --stacktrace"
```

I think it could be simplified:

```suggestion
    "build": "antora ../antora-playbook.yml --stacktrace"
```

In practice, it will be equivalent if you installed the dependencies.

---

Oops! I copied the wrong one... you already convinced me this works.